### PR TITLE
Adds HoP and Alert Levels entry, cleans up other entries

### PR DESCRIPTION
**Pull request by UBlueberry**
## About the PR

HoP has their own entry explaining their job, how to change IDs and how to replace equipment using their PTech. Also adds a bunch of links to the antagonist entries. Also also cleans up the character creation entry.
## Why / Balance

Helps ease the learning curve also takes some information from the wiki and puts it in the guidebook
## Media

image
image
image
image
image
image
image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/ServerInfo/Guidebook/ReferenceTables/AlertLevels.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    [italic]Attention! Station alert level is now green! It is now safe to return to your workplaces.[/italic]
+
+    [color=green]Code Green[/color] signifies that there is [bold]no danger[/bold] to the station or its crew. Personnel should:
+    - Work as usual
+    - Partake in recreational activities
+    - Enjoy it while it lasts
+
+    ## Code Blue
+    [italic]Attention! Station alert level is now Blue! There is a confirmed threat to the station. Security should perform random checks. Crewmembers are advised to be vigilant and report suspicious activity to security.[/italic]
+    <!-- this color isn't the same as the alert color because it's easier(?) to read -->
```
Comment (Aeshus) [inline Resources/ServerInfo/Guidebook/ReferenceTables/AlertLevels.xml:15]: You can't use html comments in guidebook entries
```suggestion
```

File: Resources/ServerInfo/Guidebook/ReferenceTables/AlertLevels.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    [italic]Attention! Station alert level is now Red! There is an immediate threat to the station. Security should prepare to use lethal force if necessary. Crewmembers should find a safe place to shelter in, and are advised to follow any present authorities.[/italic]
+
+    [color=red]Code Red[/color] warns personnel that there is an [bold]active danger[/bold] to the crew or the vessel. Personnel should:
+    - Assess the situation and choose between sheltering in place or evacuating dangerous areas
+    - Work as usual if sheltering in place
+    - Comply with Security and Command
+
+    ## Code Violet
+    [italic]Attention! Station alert level is now Violet! There is a viral threat on the station. Medical staff are advised to isolate crewmembers with any symptoms. Crewmembers are advised to distance themselves from others and perform safety measures to prevent further spread.[/italic]
+    <!-- this needs to be updated when virology comes back -->
```
Comment (Aeshus) [inline Resources/ServerInfo/Guidebook/ReferenceTables/AlertLevels.xml]: Can't use this one either
```suggestion
```

File: Resources/ServerInfo/Guidebook/ReferenceTables/AlertLevels.xml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<Document>
+    # Alert Levels
+    Alert levels can be set at communcations consoles.
```
Comment (ilyamikcoder) [inline Resources/ServerInfo/Guidebook/ReferenceTables/AlertLevels.xml:3]:
```suggestion
    Alert levels can be set at communications computers.
```

typo. also it's called `communications computer` (see [here](https://github.com/space-wizards/space-station-14/blob/9b40bcc1d9cd63938ccd9833628c23761feef456/Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml#L560C9-L560C32))

**Comment by UBlueberry** [comment]
gonna split this into multiple prs to make this easier on everyone

**Comment by UBlueberry** [comment]
gonna split this into multiple prs to make this easier on everyone

**Pull request closed without merging**